prow/oss/cluster/gerrit.yaml

```diff
@@ -26,6 +26,7 @@ spec:
         - --job-config-path=/etc/job-config
         - --cookiefile=/etc/cookies/cookies
         - --gerrit-projects=https://kunit-review.googlesource.com=linux
```

Comment:
is this still relevant? aka is this repo still under `kunit`?

Reply:
Yep, we still use the kunit GoB instance for KUnit specific stuff. linux-review is a new Gerrit instance that we share with another team for the purpose of upstream testing (not KUnit specific stuff).